main.roc

```diff
@@ -0,0 +1,80 @@
+app "generate-build"
+    packages { pf: "/home/username/gitrepos/roc/examples/static-site-gen/platform/main.roc" }
```

Comment:
Similar problem in reverse for me. I guess in future we will be able to have a release of static-site somewhere we can update this link to? I'm happy to log an issue here to track this.